scripts/run_tests.sh

```diff
@@ -6,10 +6,14 @@ set -e
 (cd /app/relation_engine_server/test/spec_release && \
   tar czvf spec.tar.gz sample_spec_release)
 
+echo "> black"
```

Comment:
Curious why not `set -x`

Reply:
The tests do fail already if one of these fails, it was just sometimes hard to tell which of the 4 different checkers was failing